packages/lab/src/dialog/Dialog.tsx

```diff
@@ -55,83 +68,91 @@ export const Dialog = forwardRef<HTMLDivElement, DialogProps>(function Dialog(
   const {
     children,
     className,
-    open = true,
+    open = false,
     onOpenChange,
     status,
-    initialFocus,
+    disableDismiss,
+    size = "medium",
+    id: idProp,
     ...rest
   } = props;
-  const dialogId = useId() || "dialog";
   const targetWindow = useWindow();
   useComponentCssInjection({
     testId: "salt-dialog",
     css: dialogCss,
     window: targetWindow,
   });
 
+  const currentbreakpoint = useCurrentBreakpoint();
+
   const [showComponent, setShowComponent] = useState(false);
 
-  const { floating, context, getFloatingProps } = useDialog({
+  const { context, floating } = useFloatingUI({
     open,
     onOpenChange,
   });
 
+  const id = useId(idProp);
+
+  const { getFloatingProps } = useInteractions([
+    useClick(context),
+    useRole(context, { role: "dialog" }),
```

Comment:
Should this be changed to render 'alertdialog' when alert dialog is passed as the role, or is it fine to have dialog and alert dialog both attached as roles ?
Could alternatively advise to pass down aria-role='alert dialog'
Any thoughts ?

Reply:
This should be configurable

Reply:
no we don't - because it comes as a feature of use interactions it felt easier to use their implementation.
Will change so it is configurable